Packs/DigitalGuardian/Integrations/DigitalGuardianARCEventCollector/DigitalGuardianARCEventCollector_description.md

```diff
@@ -1,18 +1,13 @@
 ## Digital Guardian ARC Event Collector Help
 
-### Required Parameters
+**Important:** 
+This integration is supported by Palo Alto Networks. 
 
-API Client ID 
+### Configuration
+Event fetch interval should be set to "1 hour".
+System fails when fetching every "1 minute".
```

Comment:
Let's add this to a new section in the integration README - "know limitations". See other integration in xsoar.pan